test/compress/collapse_vars.js

```diff
@@ -588,12 +587,11 @@ collapse_vars_lvalues: {
         function f2(x) { var z = x, a = ++z; return z += a; }
         function f3(x) { var a = (x -= 3); return x + a; }
         function f4(x) { var a = (x -= 3); return x + a; }
-        function f5(x) { var w = e1(), v = e2(), c = v = --x; return (w = x) - c; }
```

Comment:
Can you please set `unused: 2` in the collapse_vars tests with different results?